interfaces/builtin/gpio_chardev.go

```diff
@@ -206,7 +206,7 @@ func init() {
 			name:                     "gpio-chardev",
 			summary:                  gpioChardevSummary,
 			baseDeclarationSlots:     gpioChardevBaseDeclarationSlots,
-			connectedSlotKModModules: gpioChardevConnectedSlotKmod,
+			permanentPlugKModModules: gpioChardevPermanentPlugKmod,
```

Comment:
would it make more sense to associate with the slot? i.e. the gadget being the sort of provider